apps/wow-docs/app/layout.tsx

```diff
@@ -20,9 +51,10 @@ const RootLayout = ({
         <Sidebar />
         <styled.main
           height="100vh"
+          left="180px"
           padding="70px 102px 0 101px"
           position="relative"
```

Comment:
P2;
`position`이 `relative`이기 때문에 `left` 속성이 안먹는 것 같아용!
지금 `main` 태그 안에 존재하는 레이아웃 스타일이 조금 틀어져서 전반적으로 가운데 정렬이 안맞는 것 같은데, 

아래와 같이 스타일 바꿔보면 조금 더 좋아질 것 같아요.

```suggestion
<styled.main
    height="100vh"
    padding="70px 102px 0 101px"
    position="relative"
    width="100%"
    margin-left="250px"
>
```

Reply:
요것도 반영했숩니다~